<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SliderSceneItemEdit</class>
 <widget class="QWidget" name="SliderSceneItemEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SliderSceneItemEdit</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Slider Ayarları</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Slider Tipi : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxSliderType">
          <item>
           <property name="text">
            <string>Dikey</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Yatay</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Dikey Ters</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Yatay Ters</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Minimum Değer :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditMinSliderVal"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Maximum Değer :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditMaxSliderVal"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Slider Kalınlığı : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSliderSliderKalinligi">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>512</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditKalinlik"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Slider uzunluğu:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSliderSliderLength">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>1024</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditSliderLength"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Slider Genel Css</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEditCss"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Min</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSliderMinMaxDeneme">
          <property name="pageStep">
           <number>1</number>
          </property>
          <property name="sliderPosition">
           <number>99</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Max</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelPercentage">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
